backend/metering_billing/models.py

```diff
@@ -1476,6 +1476,10 @@ def calculate_revenue(self, usage: float, prev_tier_end=False):
         )
         # for usage in usage_dict.values():
         usage = convert_to_decimal(usage)
+        
+        if bulk_pricing_enabled and self.range_end is not None and self.range_end < usage:
```

Comment:
Should include this range. In teh case of tiers from 0-50, 50-100, want to use the 50-100 price. In this case exactly 50 would cause a bug

Reply:
@meehawk Is this fixed?